### DISCUSSION TITLE
Saleor installation issue

### DISCUSSION THREAD

**Discussion by foobin1** [saleor/saleor#8711] [Deployment]
I compiled the latest version of saleor and when loading saleor it raises the CPU load very high, like 80-90% load. So much so it crashes the server and need to force restart to ssh back in. It seems its the node.js service that is maxing out the CPU. Anyone know how to fix this issue? I followed the installation instructions on the website without changing any other parameters. Thanks

**Comment by karolkielecki** [comment]
@foobin1 Hi! Could you please provide more info? Are you using the latest beta version?

**Comment by foobin1** [comment]
Hello Karol,

Yes i am using the latest version.

Is there a more stable version i should be using? Thanks.

**Comment by krzysztofwolski** [comment]
Did you deployed whole platform as is? If so, it is not recommended to use it this way because frontend servers are launched in dev mode (not optimized + debugging tools enabled).
Which node services produces such load, is it storefront or dashboard?
